specification/ai/ModelClient/models/chat_completions.tsp

```diff
@@ -201,6 +218,24 @@ model CompletionsUsage {
   total_tokens: int32;
 }
 
+@doc("A breakdown of tokens used in a completion.")
+model CompletionsUsageDetails {
+  @doc("The number of tokens corresponding to audio input.")
```

Comment:
I don't remember this being in Facundo's PR. Did he indicate this CompletionsUsageDetails is/will be part of the common REST APIs?